CONTRIBUTING.md

```diff
@@ -93,6 +93,16 @@ The process of proposing a change to Talawa Admin can be summarized as:
          npm install --legacy-peer-deps
          npm run test --watchAll=false --coverage
       ```
+      
+      2. Debug Tests in browser 
+      ```
+         npm install --legacy-peer-deps
+         npm run jest-preview
+         npm run test --watchAll=false --coverage
+      ```
+      
+      The preview for failed tests will be visible at http://localhost:3336 
```

Comment:
There must be more to this. For example, wouldn't this fail if the `npm run jest-preview` command is run again because there would already be a listener on port 3336? You'll need to explain how to terminate this between testing sessions

Reply:
I've made the docs more descriptive to address this issue ✅